donut/modules/groups/routes.py

```diff
@@ -18,6 +18,10 @@ def get_groups_list():
         fields = [f.strip() for f in flask.request.args["fields"].split(',')]
     return jsonify(helpers.get_group_list_data(fields=fields, attrs=attrs))
 
+@blueprint.route("/1/positions/")
+def get_positions():
+    return jsonify(result = helpers.get_position_data())
```

Comment:
We're using a more recent version of jsonify which doesn't require "result=". pip upgrade flask.